### PR TITLE
Upload SQL Database for 'Suggestions' section.

### PR DESCRIPTION
**Pull request by Ahbe13**
## Checklist
- [x] Include 3 type's of suggestions: Books, Anime, Games.
- [x] Include 3 genre's per suggestion type.
- [x] Include 4 suggestions per genre.
- [x] Include all relevant information needed such as price, name, description. 
- [x] Manually tested to ensure the database works correctly.

### CODE REVIEW

File: Genre content DB.sql [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    );
+
+CREATE TABLE Anime ( 
+	Anime_ID INTEGER PRIMARY KEY NOT NULL,
+	Anime_Name VARCHAR(50) UNIQUE NOT NULL,
+    Anime_Genre VARCHAR(25),
+	Where_TW VARCHAR(25), 
+    Anime_Script VARCHAR(1000) 
+    );
+
```
Comment (Meteorstrike1) [inline Genre content DB.sql:20]: Test

File: Genre content DB.sql [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+(9, 'STARDEW VALLEY', 'SIMULATION', 'PC, NINTENDO SWITCH, XBOX, PLAYSTATION', 10.99, 'Youve inherited your grandfathers old farm plot in Stardew Valley. Armed with hand-me-down tools and a few coins, you set out to begin your new life. Can you learn to live off the land and turn these overgrown fields into a thriving home?'),
+(10, 'Two Point Hospital', 'SIMULATION', 'PC, NINTENDO SWITCH, XBOX, PLAYSTATION', 24.99, 'Design stunning hospitals, cure peculiar illnesses and manage troublesome staff as you spread your budding healthcare organisation across Two Point County.'),
+(11, 'GAME DEV TYCOON', 'SIMULATION', 'PC', 8.50, 'In Game Dev Tycoon you replay the history of the gaming industry by starting your own video game development company in the 80s. Create best selling games. Research new technologies and invent new game types. Become the leader of the market and gain worldwide fans.'),
+(12, 'NAHEULBEUKS DUNGEON MASTER', 'SIMULATION', 'PC', 20.99, 'A dungeon in danger ! Build, manage, and defend your tower in the satirical heroic fantasy universe of Dungeon of Naheulbeuk. From a shaky establishment to an infamous lair!');
+
```
Comment (Meteorstrike1) [inline Genre content DB.sql:77]: Looks really good I like the amount the options

**Comment by Meteorstrike1** [comment]
I'm not sure if it is possible or not (getting conflicting answers) but can you change the destination of the merge from main to development? We're going to try and keep all the changes in development branch for now.

**Comment by Meteorstrike1** [comment]
Actually was just looking through the tables again, I'm not 100% sure on which fields are NOT NULL and which aren't. I think genre and anything else that is important getting the result generated should be NOT NULL as well? And maybe price could be optional.

**Comment by Angel2001-programmer** [comment]
Hi @Meteorstrike1 
Just changed the PR from main to development branch. 
I hope this helps!

**Comment by Ahbe13** [comment]
> Actually was just looking through the tables again, I'm not 100% sure on which fields are NOT NULL and which aren't. I think genre and anything else that is important getting the result generated should be NOT NULL as well? And maybe price could be optional.

I will make changes to the 'NOT NULL' tomorrow, just wanted you to see what I have done so far

File: Genre content DB.sql (hunk 2 above)
Comment (Ahbe13) [reply Genre content DB.sql:77]: Totally understandable I just picked some that I have played myself and random ones off of steam seems as we only need them for examples. Ideally if we were to make the entire website functional we would have 1000s of genre's and titles coming from an public API and wouldn't need a database for that at all. I will make note of this to mention in the documentation so our marker is aware

File: Genre content DB.sql (hunk 1 above)
Comment (Ahbe13) [reply Genre content DB.sql:20]: Yeah I think at this point we dont really need to worry about age ratings, but mentioning the possibility of doing so in the documentation is a good idea

**Pull request merged**